src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Arrays;
+
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball[] balls = {lottery.getRandomBall(), lottery.getRandomBall(), lottery.getRandomBall()};
+        System.out.println(Arrays.toString(balls));
```

Comment:
it's an interesting approach, but let's use a for loop for this, as mentioned in the [checklist](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery)
![image](https://github.com/mate-academy/jv-lottery/assets/49767049/b2461e52-9d64-4808-83ad-83bf76f60ec1)